lib/connection/process/basic2.js

```diff
@@ -57,6 +57,7 @@ function getUnix (path, args, env) {
             ty: ty,
             kill: () => ty.kill(),
             interrupt: () => ty.kill('SIGINT'),
+            interruptREPL: () => ty.write('\x03'),
```

Comment:
Why do we need this?

Reply:
We don't really except for the case of running on windows without the powershell wrapper.

Reply:
Ok cool. I think a quick note here would be useful so no-one removes it by accident